applications/equitypricemodel/src/equitypricemodel/combine_data.py

```
@@ -0,0 +1,47 @@
+        "volume_weighted_average_price",
+        "sector",
+        "industry",
+    )
+
+    filtered_data = consolidated_data.select(retained_columns)
+
+    filtered_data.write_csv(output_csv_path)
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 4:  # noqa: PLR2004
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)